alertwildfire_live_recording.py

```diff
@@ -223,7 +228,38 @@ def main():
             result = pool.map(fetchAllCameras, camera_bunchs)
             pool.close()
     else:
-        fetchAllCameras(listofRotatingCameras)
+        if args.full_system:
+            response_time_per_camera = test_System_response_time(trial_length = 10)
+            listofCameras = alertwildfire_API.get_all_camera_info()
+            target_refresh_time_per_camera, listofTargetCameras, num_of_processes_needed, num_cameras_per_process, num_of_agents_needed = {},{},{},{},0
+            # target estimate of camera refresh time
+            target_refresh_time_per_camera["rotating"]   = 12#secs
+            target_refresh_time_per_camera["stationary"] = 60#secs
+            #separation of data by type
+            listofTargetCameras["rotating"] = [camera["name"] for camera in listofCameras if (camera["name"][-1]=='2') ]
+            listofTargetCameras["stationary"] = [camera["name"] for camera in listofCameras if (camera["name"][-1]!='2') ]
+            camera_bunchs= []
+            for type in ["rotating","stationary"]:
+                num_cameras_per_process[type] = math.floor(target_refresh_time_per_camera[type] / response_time_per_camera)
+                #divy up cameras rotating and stationary to maximize efficiency
+                num_of_processes_needed[type] =  math.ceil(len(listofTargetCameras[type])/num_cameras_per_process[type])
+                num_cameras_per_process[type] = math.floor(len(listofTargetCameras[type])/num_of_processes_needed[type])
+                num_of_agents_needed += num_of_processes_needed[type]
+                for num in range(0, num_of_processes_needed[type]):
+                    split_start = num_cameras_per_process[type]*num
+                    split_stop = num_cameras_per_process[type]*num+num_cameras_per_process[type]
+                    camera_bunchs.append(listofRotatingCameras[type][split_start:split_stop])
+
+            with Pool(processes=num_of_agents_needed) as pool:
```

Comment:
It would be good to print out the numbers and possibly place a limit on the max number of agents.